clusterloader2/pkg/measurement/measurementfactory.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package measurement
+
+import (
+	"fmt"
+	"sync"
+)
+
+var factory *measurmentsFactory
```

Comment:
spelling mistake

Reply:
Also change it to `measurementFactory` (i.e plural isn't needed).

Reply:
done